cirrus: freebsd 12.1 tests fail because of bison

As @dominicjaeger reported in #3649, the FreeBSD 12.1 and FreeBSD ASAN (also 12.1) build jobs fail.

```
CMake Error at /usr/local/share/cmake/Modules/FindBISON.cmake:103 (message):
  Command "/usr/local/bin/bison --version" failed with output:
  ld-elf.so.1: /usr/local/bin/bison: Undefined symbol
  "fputs_unlocked@FBSD_1.6"
Call Stack (most recent call first):
  src/plugins/toml/CMakeLists.txt:17 (find_package)
  src/plugins/toml/CMakeLists.txt:105 (check_dependencies)
```

A quick google search revealed that this can happens after an incomplete base update of freebsd. Since this only happens on 12.1 and 12.1 has already ended its support. Maybe updating the 12.1 images to 12.2 (latest version) fixes this issue?

## System Information

- Elektra Version: master
- Cirrus FreeBSD 12.1

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -37,7 +37,7 @@ bsd_task:
 
     - name: 😈 ASAN
       freebsd_instance:
-        image_family: freebsd-12-2
+        image_family: freebsd-12-1-snap
       env:
         ENABLE_ASAN: ON
 

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -53,13 +53,12 @@ bsd_task:
       ninja
       yajl
       yaml-cpp
-    - echo "CC=clang11" >> /etc/make.conf
-    - echo "CXX=clang++11" >> /etc/make.conf
-    - echo "CPP=clang-cpp11" >> /etc/make.conf
 
   script:
     - > # We use `-std=c11`, since the header `math.h` on FreeBSD requires C11 features
       set --
+      -DCMAKE_C_COMPILER=clang11
+      -DCMAKE_CXX_COMPILER=clang++11
       -DBINDINGS='ALL;-io_glib'
       -DINSTALL_SYSTEM_FILES=ON
       -DCMAKE_SKIP_INSTALL_RPATH=ON

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -37,7 +37,7 @@ bsd_task:
 
     - name: 😈 ASAN
       freebsd_instance:
-        image_family: freebsd-12-2
+        image_family: freebsd-13-0
       env:
         ENABLE_ASAN: ON
 

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -33,11 +33,11 @@ bsd_task:
 
     - name: 😈 FreeBSD 12
       freebsd_instance:
-        image_family: freebsd-12-1
+        image_family: freebsd-12-2
 
     - name: 😈 ASAN
       freebsd_instance:
-        image_family: freebsd-12-1
+        image_family: freebsd-12-2
       env:
         ENABLE_ASAN: ON
 

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -37,7 +37,7 @@ bsd_task:
 
     - name: 😈 ASAN
       freebsd_instance:
-        image_family: freebsd-12-1-snap
+        image_family: freebsd-12-2
       env:
         ENABLE_ASAN: ON
 

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -37,7 +37,7 @@ bsd_task:
 
     - name: 😈 ASAN
       freebsd_instance:
-        image_family: freebsd-13-0
+        image_family: freebsd-12-2
       env:
         ENABLE_ASAN: ON
 
@@ -49,9 +49,13 @@ bsd_task:
       flex
       git
       libgit2
+      llvm11
       ninja
       yajl
       yaml-cpp
+    - echo "CC=clang11" >> /etc/make.conf
+    - echo "CXX=clang++11" >> /etc/make.conf
+    - echo "CPP=clang-cpp11" >> /etc/make.conf
 
   script:
     - > # We use `-std=c11`, since the header `math.h` on FreeBSD requires C11 features

--- a/doc/news/_preparation_next_release.md
+++ b/doc/news/_preparation_next_release.md
@@ -148,7 +148,7 @@ you up to date with the multi-language support provided by Elektra.
 
 ### Cirrus
 
-- <<TODO>>
+- Update FreeBSD images from version 12.1 to 12.2 _(Robert Sowula)_
 - <<TODO>>
 - <<TODO>>
 

